Jenkinsfile

```diff
@@ -81,11 +81,8 @@ def getTestPlan() {
   if (params.e2e_continuous == true)  {
     return xray_continuous_testplan
   }
-  def causes = currentBuild.getBuildCauses()
-  for(cause in causes) {
-    if ("${cause}".contains("hudson.triggers.TimerTrigger\$TimerTriggerCause")) {
-      return xray_nightly_testplan
-    }
+  if (isTimed() == true) {
```

Comment:
should this also be `if (isTimed() == true || params.run_as_nightly == true) ` ?

Reply:
Only if we want to send the results to the nightly test plan. I thought about this and thought that it was inappropriate for this use case.